### ISSUE TITLE
IFC not rendering correctly

### ISSUE THREAD

**Issue by vegarringdal** [tomvandig/web-ifc#27]
I can send sample files on discord in a PM

**With web-ifc I get this:**
1
![image](https://user-images.githubusercontent.com/2901416/117673617-3a442300-b1ab-11eb-872d-bc38933d4948.png)

2 (overlook the red color)
![image](https://user-images.githubusercontent.com/2901416/117674854-53999f00-b1ac-11eb-89dd-b5a5e78d536f.png)

**In blender:**
1
![image](https://user-images.githubusercontent.com/2901416/117673202-dcafd680-b1aa-11eb-87c3-df9b5ffc345b.png)
2
![image](https://user-images.githubusercontent.com/2901416/117674723-349b0d00-b1ac-11eb-8487-c78fc84bc26f.png)

**Comment by tomvandig** [comment]
Just implemented the missing component, result can be tested here: https://tomvandig.github.io/web-ifc/examples/viewer/index.html

To make the model visible, I also had to solve a coordination issue, so models are now default coordinated on the first element found. This will have to change in the future.

Your second model is actually rendering "correctly", however the default number of segments in a circle is currently set to 5, which I admit is a bit of a loose definition of circle. Will make this option configurable in the future.

**Comment by vegarringdal** [comment]
@tomvandig Awesome work. 👍 🥇 

> To make the model visible, I also had to solve a coordination issue, so models are now default coordinated on the first element found. This will have to change in the future.

About the coordinates, dont think about it, I did a update to viewer repo that will focus model but its not set a default behavior behavior atm. 
https://github.com/agviegas/web-ifc-viewer/blob/master/viewer/src/core/viewer.ts#L133-L155


Closing this issue.
If I find any new bugs Ill create a new issue and send you model files on discord.
